ocs_ci/ocs/monitoring.py

```diff
@@ -56,46 +63,64 @@ def validate_pvc_are_mounted_on_monitoring_pods(pod_list):
 
     Args:
         pod_list (list): List of the pods where pvc are mounted
+
     """
     for pod in pod_list:
         pod_obj = get_pod_obj(
-            name=pod, namespace='openshift-monitoring'
+            name=pod.name, namespace=defaults.OCS_MONITORING_NAMESPACE
         )
         mount_point = pod_obj.exec_cmd_on_pod(command="df -kh")
-        assert "/dev/rbd" in mount_point, f"pvc is not mounted on pod {pod}"
+        assert "/dev/rbd" in mount_point, f"pvc is not mounted on pod {pod.name}"
     logger.info("Verified all pvc are mounted on monitoring pods")
 
 
-def validate_monitoring_pods_are_respinned_and_running_state(pods_list):
-    """
-    Validate monitoring pods are respinned and running state
-
-    Args:
-        pod_list (list): List of the pods where pvc are mounted
-    """
-    ocp = OCP(api_version='v1', kind='Pod', namespace='openshift-monitoring')
-    assert ocp.wait_for_resource(
-        condition=constants.STATUS_PENDING, resource_name=pods_list[0]
-    ), f"failed to reach pod {pods_list[0]} "
-    f"desired status {constants.STATUS_PENDING}"
-    for pod in pods_list:
-        assert ocp.wait_for_resource(
-            condition=constants.STATUS_RUNNING, resource_name=pod
-        ), f"failed to reach pod {pod} "
-        f"desired status {constants.STATUS_RUNNING}"
-
-
 def get_list_pvc_objs_created_on_monitoring_pods():
     """
     Returns list of pvc objects
+
     """
-    pvc_list = get_all_pvcs(namespace='openshift-monitoring')
+    pvc_list = get_all_pvcs(namespace=defaults.OCS_MONITORING_NAMESPACE)
     ocp_pvc_obj = OCP(
-        kind=constants.PVC, namespace='openshift-monitoring'
+        kind=constants.PVC, namespace=defaults.OCS_MONITORING_NAMESPACE
     )
     pvc_obj_list = []
     for pvc in pvc_list['items']:
         pvc_dict = ocp_pvc_obj.get(resource_name=pvc.get('metadata').get('name'))
         pvc_obj = PVC(**pvc_dict)
         pvc_obj_list.append(pvc_obj)
     return pvc_obj_list
+
+
+def get_kube_pod_spec_volumes_persistentvolumeclaims_info_metric():
```

Comment:
pls also address comments mentioned in https://github.com/red-hat-storage/ocs-ci/pull/518 wherever applicable.